spec/person_spec.rb

```diff
@@ -0,0 +1,25 @@
+require './person'
+require './book'
+require 'date'
+
+describe Person do
+  let(:person) { Person.new(20, 'John Doe', parent_permission: true) }
+  let(:book) { Book.new('The Great Gatsby', 'F. Scott Fitzgerald') }
+  let(:rental_date) { Date.today }
+
+  describe '#initialize' do
+    it 'initializes a person with the provided age' do
+      expect(person.age).to eq(20)
+    end
+
+    it "initializes a person with the provided name or 'unknown'" do
+      expect(person.name).to eq('John Doe')
+    end
+  end
+
+  describe '#correct_name' do
+    it 'returns the correct name of the person' do
+      expect(person.correct_name).to eq('John Doe')
+    end
+  end
+end
```

Comment:
- [ ] Good job so far in writing a few tests for this file, but you are missing out on some other methods present in the person.rb file that you're required to write tests for. According to the requirement, you're expected to write tests for all methods present in all the classes. Kindly add some more tests for the add_rental, can_use_services, and of_age methods that are in the person.rb file to correspond things as expected 👍🏽

Reply:
fixed